deps/picotls/include/picotls.h

```diff
@@ -135,7 +136,7 @@ extern "C" {
 
 /* negotiated_groups */
 #define PTLS_GROUP_SECP256R1 23
-#define PTLS_GROUP_NAME_SECP256R1 "secp256r1"
+#define PTLS_GROUP_NAME_SECP256R1 "scep256r1"
```

Comment:
Typo?

Reply:
oh yeah thanks! and it's from https://github.com/jedisct1/picotls/blob/boring2/include/picotls.h#L139

Reply:
That was fixed in 8ef127d2695a9772849c2aae6df17268c96651e3 - Need to rebase the `boring2` branch.

Reply:
yeah just needs a refresh 👍🏻